lib/test_lib.py

```diff
@@ -571,7 +571,7 @@ def SetupClients(self, nr_clients):
         fd.Flush()
 
         index.AddClient(fd)
-
+    index.Close()
```

Comment:
Wouldn't the index still be missing all the clients using Open instead of Create?